lazyflow/operator.py

```diff
@@ -182,6 +203,17 @@ class Operator(metaclass=OperatorMetaClass):
     description = ""
     category = "lazyflow"
 
+    _current_transaction = None
```

Comment:
This is supposed to work this way:
1. If there is no initialized transaction on class, then `self._current_transaction` will be retrieved as a class variable
2. Otherwise, we will set an instance variable on the operator, which will shadow the class level variable
Idea was to have a universal interface and avoid `getattr` calls checking conformity to this interface (operator always reliably provide `_current_transaction`)

Reply:
Wow, I didn't know it would get shadowed rather than overwritten o.O
But is there any Operator that doesn't ultimately inherit from the `Operator` base class that would justify not having `self._current_transaction = None` in Operator's `__init__`? Or in `__new__`, for that matter, since we're already doing that crazy slot replication magic anyway...

Reply:
It's more common to have this type of default values on class level and if we at some point will have type annotations they will be also on the class level so we'll anyway have this variable there.